programmer-skills/start-to-develop-js.md

```diff
@@ -15,7 +15,7 @@ which let you execute JavaScript in the command line or like a server language
 in substitution of *php*, *java*, *ruby*, or others.
 
 ## Code Editors and IDEs
-In order to start coding, we will need a tool to write our code within. These should should help us to be more productive writing code, avoid mistakes and highlight
+In order to start coding, we will need a tool to write our code within. These should help us be more productive while writing code, avoid mistakes and highlight
```

Comment:
object of previous clause is singular "tool". following clause, these should be replaced with "This" to reference that it is a singular "tool". My recommendation, with proper punctuation: 

> "This tool should help us be more productive when writing code; avoid mistakes; and make use of proper highlighting to expose any potential syntax errors"